src/pages/Validators/constants.tsx

```diff
@@ -0,0 +1,42 @@
+import {Stack, Typography, Skeleton} from "@mui/material";
+import React from "react";
+import MetricSection from "./Components/MetricSection";
+
+export const fontSizeBody = 14;
+export const fontSizeBodySmall = 12;
+export const fontSizeSubtitle = 17;
+export const fontSizeSubtitleSmall = 15;
+export const fontSizeTitle = 25;
+export const fontSizeTitleSmall = 15;
+export const epochSectionSkeleton = (
```

Comment:
if you're trying to avoid having too many files, this component `EpochSectionSkeleton` can live in `EpochSection.tsx`.  This way `EpochSection()` can simply take `isLoading` as a prop, and render skeleton or the actual component based on `isLoading`. This can also encapsulate everything about EpochSection within one file/component.